packages/trader/src/Assets/Trading/Categories/accumulator-description-video.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { getUrlBase, isMobile } from '@deriv/shared';
+import { useStore } from '@deriv/stores';
+
+const AccumulatorDescriptionVideo = ({ selected_contract_type }: { selected_contract_type: string }) => {
+    const { ui } = useStore();
+    const { is_dark_mode_on: is_dark_theme } = ui;
+    const [is_loading, setIsLoading] = React.useState(true);
+    const getVideoSource = React.useCallback(
+        (extension: 'mp4' | 'webm') => {
+            return getUrlBase(
+                `/public/images/common/accumulator_description${is_dark_theme ? '_dark' : '_light'}.${extension}`
```

Comment:
I think it's better to add a map or object containing trade-type, e.g accumulator and the url base. To make it more extendable.